### PR TITLE
nobility has arrived

### PR DESCRIPTION
**Pull request by cornercube**
## About The Pull Request
new pilgrim class: the noble

pure RP class
starts with more coin, nicer clothing, and the ability to read
but has next to combat skills and essentially zero gamer gear
go toady up to the king or something
main color: purple

noblemen have a tiny bit of swords and bow skills to make them less bad in combat (first you gotta find one)
they also have a bit more knife fighting skill to make better use of their dagger

noblewomen have a bit of cooking, crafting, sewing, and music, to make them perfect companions
they also start with a pouch of crackers

old nobles have a wooden staff and a bit of polearm skills to use it

other stuff added:
- purple cloak
- purple shirt
- purple pants
- a subtype of pouch with some crackers in it

## Why It's Good For The Game
flavor class
diametric opposite of peasant

### CODE REVIEW

**Comment by Autisem** [comment]
maby give them some more utility skills? nobles in theory are pretty well educated compared to most people. tho not sure why noble would pilgrim but thats not too importent

**Comment by cornercube** [comment]
> maby give them some more utility skills? nobles in theory are pretty well educated compared to most people. tho not sure why noble would pilgrim but thats not too importent

in case you wanted to play as a foreign noble who is visiting
go make up a title for yourself and hope the king takes your word for it

File: code/modules/jobs/job_types/roguetown/adventurer/types/pilgrim/noble.dm [hunk 1]
```diff
@@ -0,0 +1,52 @@
+/datum/advclass/noble
+	name = "Noble"
+	allowed_sexes = list("male", "female")
+	allowed_races = list("Humen",
+	"Humen",
```
Comment (BadAtThisGame302) [inline code/modules/jobs/job_types/roguetown/adventurer/types/pilgrim/noble.dm:5]: Humen, Humen, should fix this.

Comment (Kush1Push1) [reply code/modules/jobs/job_types/roguetown/adventurer/types/pilgrim/noble.dm:5]: It's also a cursed practice to don't use defines

**Comment by BadAtThisGame302** [comment]
You should give the class the noble trait aswell.

**Pull request merged**